css-layout-classes/styles.css

```diff
@@ -0,0 +1,11 @@
+#body {
+  display: flex;
+  flex-direction: column;
+  margin: 0 10%;
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+  margin: 0 25px;
```

Comment:
Your container class shouldn't be using `display: flex`, and it's also missing a `max-width`